lib/src/main/java/uk/gov/di/ipv/cri/passport/library/service/ServiceFactory.java

```diff
@@ -18,110 +16,106 @@
 
 public class ServiceFactory {
 
-    private final ObjectMapper objectMapper;
-    private final PassportConfigurationService passportConfigurationService;
-    private final ClientFactoryService clientFactoryService;
-    private final EventProbe eventProbe;
-    private final SessionService sessionService;
-    private final AuditService auditService;
-    private final PersonIdentityService personIdentityService;
-
-    private final DataStore<DocumentCheckResultItem> documentCheckResultStore;
-
-    /** Creates common service objects used by *both* passport lambdas */
-    @ExcludeFromGeneratedCoverageReport
-    public ServiceFactory() {
-        this.objectMapper = new ObjectMapper().registerModule(new JavaTimeModule());
-
-        this.clientFactoryService = new ClientFactoryService();
-        this.passportConfigurationService = new PassportConfigurationService(clientFactoryService);
-
-        this.eventProbe = new EventProbe();
-        this.sessionService = createSessionService(passportConfigurationService);
-        this.auditService =
-                createAuditService(
-                        passportConfigurationService, objectMapper, clientFactoryService);
-        this.personIdentityService = new PersonIdentityService(passportConfigurationService);
-
-        this.documentCheckResultStore =
-                createDocumentCheckResultStore(passportConfigurationService);
+    private static ObjectMapper objectMapper;
```

Comment:
Im not really comfortable with these static and synchronised methods. We have already seen that lambda doesnt always use threads the way we expect when under load. Can we continue to create the service factory in the handler and pass it through the classes